### PR TITLE
[SW-627] Vault creation progress labels update

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Review by pstachel — APPROVED** [review]

File: src/popup/components/MultisigVaultCreationProgress.vue [hunk 1]
```diff
@@ -3,30 +3,33 @@
...
       {{ $t('modals.creatingMultisigAccount.title') }}
     </h2>
-    <ProgressBar :progress="creatingVaultProgress" />
+    <ProgressBar :progress="progressPercentage" />
     <div class="steps-list">
       <div
-        v-for="step in steps"
-        :key="step.id"
+        v-for="(step, index) in steps"
+        :key="`step${index}`"
```
Comment (konradodo) [inline src/popup/components/MultisigVaultCreationProgress.vue:10]:
```suggestion
        :key="index"
```

File: src/popup/components/MultisigVaultCreationProgress.vue [hunk 2]
```diff
@@ -3,30 +3,33 @@
...
-          class="step-item-id"
+          class="step-number"
         >
-          {{ step.id }}
+          {{ index + 1 }}
         </div>
 
         <div class="step-item-name">
-          {{ getStepText(step) }}
+          {{ step.text }}<span v-if="isStepCurrent(index)">...</span>
```
Comment (konradodo) [inline src/popup/components/MultisigVaultCreationProgress.vue:32]: "..." ->  Hmm i think good option is create constant for this

**Review by konradodo — APPROVED** [review]

File: src/popup/components/MultisigVaultCreationProgress.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/MultisigVaultCreationProgress.vue:10]: OK

File: src/popup/components/MultisigVaultCreationProgress.vue (hunk 2 above)
Comment (peronczyk) [reply src/popup/components/MultisigVaultCreationProgress.vue:32]: I'm gonna use `&hellip;` because using JS const is a bad idea in my opinion.

**Pull request merged**